lib/cocoapods/installer/analyzer.rb

```diff
@@ -217,7 +217,7 @@ def generate_podfile_state
       def update_repositories
         sources.each do |source|
           if source.git?
-            config.sources_manager.update(source.name)
+            config.sources_manager.update(source.name, true)
```

Comment:
There was a bit of discussion about this in the original issue. Also pod setup show the progress by default?